index.html

```diff
@@ -20,7 +20,7 @@ <h1>Practicing Git</h1>
   
                 <h2>Version control is...</h2>
 
-		<p>... inferior to keeping track of projects through email attachments.</p>
+		<p>... making sure every version of the software is organized and it has the bugs or new implementations it brings.</p>

```

Comment:
Let's write this section as a full sentence or a paragraph by removing the ellipsis